### PR TITLE
helm-docs

### PR DESCRIPTION
**Pull request by pierluigilenoci**
#### Which issue this PR fixes

Fixes https://github.com/fluxcd-community/helm-charts/pull/31#issuecomment-985480385 

#### Checklist

- [x] [DCO](https://github.com/fluxcd-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Chart Version bumped
- [x] helm-docs are updated
- [ ] Helm chart is tested

### CODE REVIEW

**Comment by dwerder** [comment]
You are changing the content of the chart. Which means the chart version needs to be bumped. See the details of the failed check.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,32 @@
...
+.LSOverride
+
+### IntelliJ IDE
+.idea
+
 .work
+
+# helm chart dependencies
+flux2/charts/
+flux2-sync/charts/
```
Comment (dwerder) [inline .gitignore:31]: Why is that?

**Comment by haarchri** [comment]
@dwerder think no Change in HelmChart so why the Check is running ?

**Comment by dwerder** [comment]
As i mentioned above. Even changing the README is a change in the chart and requires a minor version bump

**Review by haarchri — APPROVED** [review]

**Comment by dwerder** [comment]
LGTM

**Pull request merged**